<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="supprimer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>gestion des ticket</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Referance</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_referance_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prix_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_destination_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_ajouter">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter ticket</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_modifier">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier ticket</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_tri">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TRI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_qr">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>QR Code</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_revenu">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Revenu </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>depart</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_depart">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>arrive</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_arrive_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>671</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../logo final.png</pixmap>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>label</zorder>
     <zorder>le_referance_2</zorder>
     <zorder>label_2</zorder>
     <zorder>le_prix_2</zorder>
     <zorder>label_3</zorder>
     <zorder>le_destination_2</zorder>
     <zorder>qb_ajouter</zorder>
     <zorder>qb_modifier</zorder>
     <zorder>qb_tri</zorder>
     <zorder>qb_qr</zorder>
     <zorder>qb_revenu</zorder>
     <zorder>label_12</zorder>
     <zorder>le_date_2</zorder>
     <zorder>label_13</zorder>
     <zorder>le_depart</zorder>
     <zorder>label_14</zorder>
     <zorder>le_arrive_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_supprimer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 195, 151);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_supprimer">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>360</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer ticket</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_tickets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2_2">
     <attribute name="title">
      <string>recherche</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>validé</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 195, 151);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>MAP</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 195, 151);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ouvrir map</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
